lib/options.js

```diff
@@ -386,6 +386,19 @@ module.exports = {
         usage: ['CONVERSION'],
         supportedIn: [VERSION20, VERSION30, VERSION31],
         supportedModuleVersion: [MODULE_VERSION.V2, MODULE_VERSION.V1]
+      },
+      {
+        name: 'Select request body type',
+        id: 'preferredRequestBodyType',
+        type: 'enum',
+        default: 'x-www-form-urlencoded',
+        availableOptions: ['x-www-form-urlencoded', 'form-data', 'raw'],
```

Comment:
Let's add 4th option which will be the default value. i.e. something like `In Order`/`default`/`fallback`.

Reply:
Added the 4th option `first-listed`, but open to your suggestion.

The ENUM options are based on the Postman UI:
<img width="593" alt="image" src="https://github.com/user-attachments/assets/c3ee7d76-fc02-4eb2-afd6-49c8af7032e2">

Reply:
Yeah, this should be good.